packages/create-plugin/templates/common/_package.json

```diff
@@ -53,6 +53,7 @@
     "ts-node": "^10.9.1",
     "tsconfig-paths": "^4.2.0",
     "typescript": "4.8.4",
+    "util": "^0.12.5",
```

Comment:
Is this package needed? I thought the text encoder and decoder were available from nodes util module?

Reply:
Hmm, I think you are actually right! I will check, thanks 👍

Reply:
Fixed in https://github.com/grafana/plugin-tools/pull/899/commits/c6eb38b0df24b7bfd77ead2493bb6bb0adf36e27.